bin/pg_repack.c

```diff
@@ -11,7 +11,7 @@
  */
 
 const char *PROGRAM_URL		= "http://reorg.github.com/pg_repack";
-const char *PROGRAM_EMAIL	= "reorg-general@lists.pgfoundry.org";
+const char *PROGRAM_EMAIL	= 0;
```

Comment:
`pg_repack --help` already checks if this is NULL, and will just skip displaying the line if there is a null email.

I'm also unfamiliar with the projects in question or what the urls used to look like, having started looking at any of them today. I guess there's web archive if I wanted to see what they looked like 3 years ago - e.g. https://web.archive.org/web/20170601090644/http://pgxnclient.projects.pgfoundry.org/ https://web.archive.org/web/20160821214550/http://lists.pgfoundry.org/pipermail/reorg-general/